src/plugins/console/public/application/containers/main/main.tsx

```diff
@@ -76,21 +103,18 @@ export function Main({ isEmbeddable = false }: MainProps) {
                   />
                 </EuiFlexItem>
                 <EuiFlexItem grow={false}>
-                  <NavIconButton
-                    iconType="keyboard"
-                    onClick={() => {}}
-                    ariaLabel={MAIN_PANEL_LABELS.shortcutsButton}
-                    dataTestSubj="consoleShortcutsButton"
-                    toolTipContent={MAIN_PANEL_LABELS.shortcutsButton}
+                  <ShortcutsPopover
+                    button={shortcutsButton}
+                    isOpen={isShortcutsOpen}
+                    closePopover={() => setIsShortcutsOpen(false)}
                   />
                 </EuiFlexItem>
                 <EuiFlexItem grow={false}>
-                  <NavIconButton
-                    iconType="questionInCircle"
-                    onClick={() => {}}
-                    ariaLabel={MAIN_PANEL_LABELS.helpButton}
-                    dataTestSubj="consoleHelpButton"
-                    toolTipContent={MAIN_PANEL_LABELS.helpButton}
+                  <HelpPopover
+                    button={helpButton}
+                    isOpen={isHelpOpen}
+                    closePopover={() => setIsHelpOpen(false)}
+                    resetTour={() => {}}
```

Comment:
I'd like to get this wired in (merging first [the onboarding pr](https://github.com/elastic/kibana/pull/191021)) before we merge this PR, otherwise we are going to have a lot of loose ends to patch up in subsequent PR's

Reply:
Sure, we can first merge the onboarding PR.